### ISSUE TITLE
[Bug Report] VFileInput does not work with Safari on MacOS or iOS

### ISSUE THREAD

**Issue by ChrisLandowski** [vuetifyjs/vuetify#7941]
### Versions and Environment
**Vuetify:** 2.0.0
**Vue:** 2.6.10
**Browsers:** Safari 12.1.1
**OS:** Mac OS 10.14.5

### Steps to reproduce
Select a file

### Expected Behavior
The filename should be displayed in the File Input component

### Actual Behavior
Nothing is in the File Input component

### Reproduction Link
<a href="https://codepen.io/anon/pen/RXaPOX?&editable=true&editors=101" target="_blank">https://codepen.io/anon/pen/RXaPOX?&editable=true&editors=101</a>

### Other comments
None of the examples at https://vuetifyjs.com/en/components/file-inputs show the selected filename when using Safari on both MacOS and iOS.

**Comment by johnleider** [comment]
@KaelWD has access to an iPad so I'm blaming this one on him

**Comment by Anton-vBR** [comment]
I experienced the same issue on Chrome (updated 5th August)

**Comment by Anton-vBR** [comment]
@nekosaur 

OS: Ipad 12.2
Browser: Chrome 76.0.3809.81

Reproduction Link (same as above)
https://codepen.io/anon/pen/RXaPOX?&editable=true&editors=101

(also tried on Iphone 7)

**Comment by tgduke** [comment]
I experience the same on my own site and in the links provided here, but if I use it on vuetify https://vuetifyjs.com/en/components/file-inputs it works fine.

**Comment by Anton-vBR** [comment]
@tgduke looking at the codepen linked from the site the Vuetify version is 2.0.7 whereas the links here are 2.0.0. I’ll try to upgrade later.

**Comment by tgduke** [comment]
@Anton-vBR I will try to downgrade to version 2.0.0 and see if it is a bug introduced in 2.0.7 or inbetween

**Comment by tgduke** [comment]
Still not working with downgrade. Maybe it is related to v-model. Trying to remove it and making a new deploy.

**Comment by sophiews-up** [comment]
Any update on this?

**Comment by jacekkarczmarczyk** [comment]
We kindly ask users to not comment on closed/resolved issues. If you believe that this issue has not been correctly resolved, please create a new [issue](https://issues.vuetifyjs.com) showing the regression or reach out to us in our [Discord community](https://community.vuetifyjs.com).

Thank you for your contribution and interest in improving Vuetify.

**Comment by 0x77dev** [comment]
Same issue here...

**Comment by MingYang2008** [comment]
This bug still exists in the latest version. Only after click the placeholder or label text, it is normal to click other places.

**Comment by 0x77dev** [comment]
Just tested it out, it works on Chrome for iOS.

**Comment by Meowzz95** [comment]
Same issue here, on iOS safari, sometimes works sometimes not

**Comment by nekosaur** [comment]
As already mentioned, please create a new issue with a reproduction in the latest version of vuetify. Thank you.
